### ISSUE TITLE
OIDC: redirect_uris for owncloud iOS app

### ISSUE THREAD

**Issue by vx3r** [authelia/authelia#2742]
### Description
ownCloud iOS app expects client `redirect_uris` to be `oc://ios.owncloud.com` according to the [doc](https://doc.owncloud.com/server/next/admin_manual/configuration/user/oidc/oidc.html#redirect-uris).
Authelia accepts only http and https in `redirect_uris`, according to the [doc](https://www.authelia.com/docs/configuration/identity-providers/oidc.html#redirect_uris) .

### Expected Behaviour
OIDC login from iOS app

### Reproduction Steps
Configure owncloud to use authelia as IDP, add default iOS client as described [here](https://doc.owncloud.com/server/next/admin_manual/configuration/user/oidc/oidc.html#owncloud-desktop-and-mobile-clients) and start login flow from iOS app

### Additional Information

Adding the `oc` uri sheme must resolve the issue, here is the full request coming from iOS app:
`GET/api/oidc/authorize?prompt=consent&response_type=code&code_challenge_method=S256&code_challenge=z7d2cS0bP6Sz03hDpeaiiJXMjQ3RgiAVfFEhVgQ4KFg&scope=openid offline_access email profile&redirect_uri=oc://ios.owncloud.com&client_id=mxd5OQDk6es5LzOzRvidJNfXLUZS2oN3oUFeXPP8LpPrhx3UroJFduGEYIBOxkY1HTTP/2.0 "Mozilla/5.0(iPhone;CPUiPhoneOS15_1likeMacOSX)AppleWebKit/605.1.15(KHTML,likeGecko)Version/15.1Mobile/15E148Safari/604.1"`

thank you

**Comment by james-d-elliott** [comment]
Marking this as a feature request as per the OIDC specification there is only mentions of http schemes, https schemes, and the urn callback. In fact several flows are *required* to use a https redirect_uri scheme. Example:

https://openid.net/specs/openid-connect-core-1_0.html#ImplicitAuthRequest

**Comment by clems4ever** [comment]
@james-d-elliott , this [RFC](https://datatracker.ietf.org/doc/html/rfc8252#section-7.1), which is the current best practice, describes private-use URI schemes so I think it's fine to consider supporting custom schemes in Authelia.

**Comment by vx3r** [comment]
Thank you all for your support on adding the feature.
It has been merged into main branch but i am still unable to start authelia with this redirect url:
```
    clients:
       -
        id: mxd5OQDk6es5LzOzRvidJNfXLUZS2oN3oUFeXPP8LpPrhx3UroJFduGEYIBOxkY1
        description: ownCloud iOS
        secret: KFeFWWEZO9TkisIQzR3fo7hfiMXlOpaqP8CFuTbSHzV1TUuGECglPxpiVKJfOXIx
        public: false
        authorization_policy: two_factor
        audience: []
        scopes:
          - openid
          - groups
          - email
          - profile
          - offline_access
        redirect_uris:
          - oc://ios.owncloud.com
        grant_types:
          - refresh_token
          - authorization_code
        userinfo_signing_algorithm: none
```
getting following error when restarting authelia:
```
authelia                      | time="2022-07-13T11:02:06+02:00" level=error msg="Configuration: identity_providers: oidc: client 'mxd5OQDk6es5LzOzRvidJNfXLUZS2oN3oUFeXPP8LpPrhx3UroJFduGEYIBOxkY1': option 'redirect_uris' has an invalid value: redirect uri 'oc://ios.owncloud.com' must have a scheme of 'http' or 'https' but 'oc' is configured"
```
with latest version:
```
❯ docker-compose images authelia
Container      Repository        Tag       Image Id       Size
----------------------------------------------------------------
authelia    authelia/authelia   latest   e189cf1abb7a   49.55 MB
```
Am i missing something here ?

thank you

**Comment by Gatherix** [comment]
@vx3r Did you ever find success with this?

**Comment by vx3r** [comment]
@Gatherix nop never worked for me

**Comment by james-d-elliott** [comment]
Feel free to open this as a discussion as it was tested by a user

**Comment by helgeklein** [comment]
I can confirm that the redirect URI `oc://android.owncloud.com` used by ownCloud's Android app works correctly with Authelia 4.37.5.

**Comment by alceasan** [comment]
> I can confirm that the redirect URI `oc://android.owncloud.com` used by ownCloud's Android app works correctly with Authelia 4.37.5.

@helgeklein, could you please post your configuration? I'm trying to get OCIS working with Authelia. So far I have been able to do it with the web client following [this guide](https://helgeklein.com/blog/owncloud-infinite-scale-with-openid-connect-authentication-for-home-networks/#sso-via-authelia-owncloud-openid-connect-authentication) but I'm having issues with the owncloud android app login. The app redirects with the web browser to my authelia site, but after a successful login there is no redirection to the app, it just stays on the "Authenticated" authelia website and never goes back to the owncloud app.

This is my authelia config for the android app:

```
      - id: e4rAsNUSIUs0lF4nbv9FmCeUkTlV9GdgTLDH1b5uie7syb90SzEVrbN7HIpmWJeD
        description: ownCloud Android app
        secret: 'dInFYGV33xKzhbRmpqQltYNdfLdJIfJ9L5ISoKhNoT9qZftpdWSP71VrpGR9pmoD'
        scopes:
          - openid
          - groups
          - profile
          - email
          - offline_access
        redirect_uris:
          - oc://android.owncloud.com
        grant_types:
          - refresh_token
          - authorization_code
        userinfo_signing_algorithm: none
```

**Comment by helgeklein** [comment]
@alceasan I've described my entire home server configuration in a very detailed series of articles. The following two articles cover Authelia:

- [Authelia & lldap: Authentication, SSO, User Management & Password Reset for Home Networks](https://helgeklein.com/blog/authentication-sso-user-management-password-reset-for-home-networks/)
- [ownCloud Infinite Scale With OpenID Connect Authentication for Home Networks](https://helgeklein.com/blog/owncloud-infinite-scale-with-openid-connect-authentication-for-home-networks/)

**Comment by C8opmBM** [comment]
Is there a working alternative for caddy?  I'm currently using SWAG (linuxserver) and I've managed with trial and error to make it work for both web and android, but I'm having issues with the desktop. 

I'm using @helgeklein method, the problematic one is the desktop. It redirects me with error to `httpS://127xxxx`, even though I have it set correctly to `http`
The error, after I delet the S in the prototcol reads (in browser)

`Error returned from the server: The provided authorization grant (e.g., authorization code, resource owner credentials) or refresh token is invalid, expired, revoked, does not match the redirection URI used in the authorization request, or was issued to another client.`

@alceasan - take a look at cors, for me this was problematic. Also I used the implicit consent, to skip the consent window.
Also make sure to use the given secrets, don't change them. Also make sure you have your authelia user created first in owncloud (first without oidc, normal login, then add your oidc).
So follow the guide @helgeklein posted. It's very detailed and works, problem is, in my case with the desktop version.

```
    cors:
      endpoints:
        - authorization
        - token
        - revocation
        - introspection
        - userinfo
      allowed_origins: "*"
      allowed_origins_from_client_redirect_uris: false
```

These work:


      - id: ownCloud-web
        description: ownCloud web client
        public: true
        authorization_policy: two_factor
        consent_mode: implicit
        redirect_uris:
          - https://cloud.xxx.net/
          - https://cloud.xxx.net/oidc-callback.html
          - https://cloud.xxx.net/oidc-silent-redirect.html
        
      - id: e4rAsNUSIUs0lF4nbv9FmCeUkTlV9GdgTLDH1b5uie7syb90SzEVrbN7HIpmWJeD
        description: ownCloud Android app
        secret: 'dInFYGV33xKzhbRmpqQltYNdfLdJIfJ9L5ISoKhNoT9qZftpdWSP71VrpGR9pmoD'
        scopes:
          - openid
          - groups
          - profile
          - email
          - offline_access
        redirect_uris:
          - oc://android.owncloud.com

This doesn't 


```
      - id: xdXOt13JKxym1B1QcEncf2XDkLAexMBFwiT9j6EfhhHFJhs2KM9jbjTmf8JBXE69
        description: ownCloud desktop client
        consent_mode: implicit
        secret: 'UBntmLjC2yYCeHwsyj73Uwo9TAaecAetRwMw0xYcvNL9yRdLSUi0hUAHfvCHFeFh'
        scopes:
          - openid
          - groups
          - profile
          - email
          - offline_access
        redirect_uris:
          - http://127.0.0.1   # I also tried http://127.0.0.1:* - this errors out. I also tried with secure/insecure settings in ocis, no fix.
          - http://localhost
```

**Comment by alceasan** [comment]
Thank you for your guide @helgeklein, I was able to deploy ocis with your config and used your Caddy config tips to get it working (the web client) with my Traefik proxy.

I already had your cors configuration in authelia @C8opmBM, it was one of the steps i had to take to get the web client working. I tried with your exact configuration, using also `consent_mode: implicit`, but to no avail. The android owncloud app just stops waiting on the authenticated webpage. This is the android application log, I can't see any error:
```
D: 2023-01-26 15:17:14:740(OwnCloudClient.java:90) .<init>()#14Creating OwnCloudClient
D: 2023-01-26 15:17:14:741(OwnCloudClient.java:128) .saveExecuteHttpMethod()Executing in request with id 5a88a80a-c9b9-4bdc-b271-2608395eeb88
D: 2023-01-26 15:17:14:814(LogBuilder.kt:38) .logHttp()[Network, request] [info] [5a88a80a-c9b9-4bdc-b271-2608395eeb88] Type: GET URL: https://ocis.xxxxx.xxx/status.php
D: 2023-01-26 15:17:14:816(LogBuilder.kt:38) .logHttp()[Network, request] [header] [5a88a80a-c9b9-4bdc-b271-2608395eeb88] X-Request-ID: 5a88a80a-c9b9-4bdc-b271-2608395eeb88
D: 2023-01-26 15:17:14:816(LogBuilder.kt:38) .logHttp()[Network, request] [header] [5a88a80a-c9b9-4bdc-b271-2608395eeb88] User-Agent: Mozilla/5.0 (Android) ownCloud-android/3.0.1
D: 2023-01-26 15:17:14:817(LogBuilder.kt:38) .logHttp()[Network, request] [header] [5a88a80a-c9b9-4bdc-b271-2608395eeb88] Accept-Encoding: identity
D: 2023-01-26 15:17:14:817(LogBuilder.kt:38) .logHttp()[Network, request] [header] [5a88a80a-c9b9-4bdc-b271-2608395eeb88] Host: ocis.xxxxx.xxx
D: 2023-01-26 15:17:14:818(LogBuilder.kt:38) .logHttp()[Network, request] [header] [5a88a80a-c9b9-4bdc-b271-2608395eeb88] Connection: Keep-Alive
D: 2023-01-26 15:17:14:818(LogBuilder.kt:38) .logHttp()[Network, request] [body] [5a88a80a-c9b9-4bdc-b271-2608395eeb88] Empty body
D: 2023-01-26 15:17:14:974(LogBuilder.kt:38) .logHttp()[Network, response] [info] [5a88a80a-c9b9-4bdc-b271-2608395eeb88] Code: 200  Message: OK IsSuccessful: true
D: 2023-01-26 15:17:14:976(LogBuilder.kt:38) .logHttp()[Network, response] [header] [5a88a80a-c9b9-4bdc-b271-2608395eeb88] Access-Control-Allow-Credentials: true
D: 2023-01-26 15:17:14:977(LogBuilder.kt:38) .logHttp()[Network, response] [header] [5a88a80a-c9b9-4bdc-b271-2608395eeb88] Access-Control-Allow-Origin: *
D: 2023-01-26 15:17:14:978(LogBuilder.kt:38) .logHttp()[Network, response] [header] [5a88a80a-c9b9-4bdc-b271-2608395eeb88] Content-Length: 276
D: 2023-01-26 15:17:14:979(LogBuilder.kt:38) .logHttp()[Network, response] [header] [5a88a80a-c9b9-4bdc-b271-2608395eeb88] Content-Security-Policy: default-src 'none';
D: 2023-01-26 15:17:14:980(LogBuilder.kt:38) .logHttp()[Network, response] [header] [5a88a80a-c9b9-4bdc-b271-2608395eeb88] Content-Type: application/json
D: 2023-01-26 15:17:14:981(LogBuilder.kt:38) .logHttp()[Network, response] [header] [5a88a80a-c9b9-4bdc-b271-2608395eeb88] Date: Thu, 26 Jan 2023 14:17:14 GMT
D: 2023-01-26 15:17:14:982(LogBuilder.kt:38) .logHttp()[Network, response] [header] [5a88a80a-c9b9-4bdc-b271-2608395eeb88] Permissions-Policy: camera 'none'; geolocation 'none'; microphone 'none'; payment 'none'; usb 'none'; vr 'none';
D: 2023-01-26 15:17:14:983(LogBuilder.kt:38) .logHttp()[Network, response] [header] [5a88a80a-c9b9-4bdc-b271-2608395eeb88] Referrer-Policy: same-origin
D: 2023-01-26 15:17:14:984(LogBuilder.kt:38) .logHttp()[Network, response] [header] [5a88a80a-c9b9-4bdc-b271-2608395eeb88] Strict-Transport-Security: max-age=63072000; includeSubDomains; preload
D: 2023-01-26 15:17:14:985(LogBuilder.kt:38) .logHttp()[Network, response] [header] [5a88a80a-c9b9-4bdc-b271-2608395eeb88] Vary: Origin
D: 2023-01-26 15:17:14:986(LogBuilder.kt:38) .logHttp()[Network, response] [header] [5a88a80a-c9b9-4bdc-b271-2608395eeb88] X-Content-Type-Options: nosniff
D: 2023-01-26 15:17:14:987(LogBuilder.kt:38) .logHttp()[Network, response] [header] [5a88a80a-c9b9-4bdc-b271-2608395eeb88] X-Download-Options: noopen
D: 2023-01-26 15:17:14:988(LogBuilder.kt:38) .logHttp()[Network, response] [header] [5a88a80a-c9b9-4bdc-b271-2608395eeb88] X-Frame-Options: allow-from https:xxxxx.xxx
D: 2023-01-26 15:17:14:989(LogBuilder.kt:38) .logHttp()[Network, response] [header] [5a88a80a-c9b9-4bdc-b271-2608395eeb88] X-Permitted-Cross-Domain-Policies: none
D: 2023-01-26 15:17:14:990(LogBuilder.kt:38) .logHttp()[Network, response] [header] [5a88a80a-c9b9-4bdc-b271-2608395eeb88] X-Robots-Tag: none,noarchive,nosnippet,notranslate,noimageindex,
D: 2023-01-26 15:17:14:990(LogBuilder.kt:38) .logHttp()[Network, response] [header] [5a88a80a-c9b9-4bdc-b271-2608395eeb88] X-Xss-Protection: 1; mode=block
D: 2023-01-26 15:17:14:992(LogBuilder.kt:38) .logHttp()[Network, response] [body] [5a88a80a-c9b9-4bdc-b271-2608395eeb88] Length: 276 byte body
D: 2023-01-26 15:17:14:992(LogBuilder.kt:38) .logHttp()[Network, response] [body] [5a88a80a-c9b9-4bdc-b271-2608395eeb88] Type: application/json
D: 2023-01-26 15:17:14:993(LogBuilder.kt:38) .logHttp()[Network, response] [body] [5a88a80a-c9b9-4bdc-b271-2608395eeb88] --> Body start for response
D: 2023-01-26 15:17:14:994(LogBuilder.kt:38) .logHttp()[Network, response] [body] [5a88a80a-c9b9-4bdc-b271-2608395eeb88] {
    "installed": true,
    "maintenance": false,
    "needsDbUpgrade": false,
    "version": "10.11.0.0",
    "versionstring": "10.11.0",
    "edition": "Community",
    "productname": "Infinite Scale",
    "product": "Infinite Scale",
    "productversion": "2.0.0-beta.5"
}
D: 2023-01-26 15:17:14:994(LogBuilder.kt:38) .logHttp()[Network, response] [body] [5a88a80a-c9b9-4bdc-b271-2608395eeb88] <-- Body end for response -- Omitted: 0 bytes
D: 2023-01-26 15:17:14:997(OwnCloudClient.java:128) .saveExecuteHttpMethod()Executing in request with id 099e3cf4-0303-4345-ae13-df720aac9ce4
D: 2023-01-26 15:17:15:001(LogBuilder.kt:38) .logHttp()[Network, request] [info] [099e3cf4-0303-4345-ae13-df720aac9ce4] Type: PROPFIND URL: https://ocis.xxxxx.xxx/remote.php/dav/files/
D: 2023-01-26 15:17:15:001(LogBuilder.kt:38) .logHttp()[Network, request] [header] [099e3cf4-0303-4345-ae13-df720aac9ce4] Depth: 0
D: 2023-01-26 15:17:15:002(LogBuilder.kt:38) .logHttp()[Network, request] [header] [099e3cf4-0303-4345-ae13-df720aac9ce4] X-Request-ID: 099e3cf4-0303-4345-ae13-df720aac9ce4
D: 2023-01-26 15:17:15:002(LogBuilder.kt:38) .logHttp()[Network, request] [header] [099e3cf4-0303-4345-ae13-df720aac9ce4] User-Agent: Mozilla/5.0 (Android) ownCloud-android/3.0.1
D: 2023-01-26 15:17:15:003(LogBuilder.kt:38) .logHttp()[Network, request] [header] [099e3cf4-0303-4345-ae13-df720aac9ce4] Accept-Encoding: identity
D: 2023-01-26 15:17:15:003(LogBuilder.kt:38) .logHttp()[Network, request] [header] [099e3cf4-0303-4345-ae13-df720aac9ce4] Content-Type: application/xml; charset=utf-8
D: 2023-01-26 15:17:15:004(LogBuilder.kt:38) .logHttp()[Network, request] [header] [099e3cf4-0303-4345-ae13-df720aac9ce4] Content-Length: 477
D: 2023-01-26 15:17:15:004(LogBuilder.kt:38) .logHttp()[Network, request] [header] [099e3cf4-0303-4345-ae13-df720aac9ce4] Host: ocis.xxxxx.xxx
D: 2023-01-26 15:17:15:005(LogBuilder.kt:38) .logHttp()[Network, request] [header] [099e3cf4-0303-4345-ae13-df720aac9ce4] Connection: Keep-Alive
D: 2023-01-26 15:17:15:005(LogBuilder.kt:38) .logHttp()[Network, request] [body] [099e3cf4-0303-4345-ae13-df720aac9ce4] Length: 477 byte body
D: 2023-01-26 15:17:15:006(LogBuilder.kt:38) .logHttp()[Network, request] [body] [099e3cf4-0303-4345-ae13-df720aac9ce4] Type: application/xml; charset=utf-8
D: 2023-01-26 15:17:15:006(LogBuilder.kt:38) .logHttp()[Network, request] [body] [099e3cf4-0303-4345-ae13-df720aac9ce4] --> Body start for request
D: 2023-01-26 15:17:15:007(LogBuilder.kt:38) .logHttp()[Network, request] [body] [099e3cf4-0303-4345-ae13-df720aac9ce4] <?xml version='1.0' encoding='UTF-8' ?><propfind xmlns="DAV:" xmlns:CAL="urn:ietf:params:xml:ns:caldav" xmlns:CARD="urn:ietf:params:xml:ns:carddav" xmlns:SABRE="http://sabredav.org/ns" xmlns:OC="http://owncloud.org/ns"><prop><displayname /><getcontenttype /><resourcetype /><getcontentlength /><getlastmodified /><creationdate /><getetag /><quota-used-bytes /><quota-available-bytes /><OC:permissions /><OC:id /><OC:size /><OC:privatelink /><OC:share-types /></prop></propfind>
D: 2023-01-26 15:17:15:007(LogBuilder.kt:38) .logHttp()[Network, request] [body] [099e3cf4-0303-4345-ae13-df720aac9ce4] <-- Body end for request -- Omitted: 0 bytes
D: 2023-01-26 15:17:15:023(LogBuilder.kt:38) .logHttp()[Network, response] [info] [099e3cf4-0303-4345-ae13-df720aac9ce4] Code: 401  Message: Unauthorized IsSuccessful: false
D: 2023-01-26 15:17:15:025(LogBuilder.kt:38) .logHttp()[Network, response] [header] [099e3cf4-0303-4345-ae13-df720aac9ce4] Access-Control-Allow-Credentials: true
D: 2023-01-26 15:17:15:026(LogBuilder.kt:38) .logHttp()[Network, response] [header] [099e3cf4-0303-4345-ae13-df720aac9ce4] Content-Length: 0
D: 2023-01-26 15:17:15:027(LogBuilder.kt:38) .logHttp()[Network, response] [header] [099e3cf4-0303-4345-ae13-df720aac9ce4] Date: Thu, 26 Jan 2023 14:17:14 GMT
D: 2023-01-26 15:17:15:028(LogBuilder.kt:38) .logHttp()[Network, response] [header] [099e3cf4-0303-4345-ae13-df720aac9ce4] Permissions-Policy: camera 'none'; geolocation 'none'; microphone 'none'; payment 'none'; usb 'none'; vr 'none';
D: 2023-01-26 15:17:15:029(LogBuilder.kt:38) .logHttp()[Network, response] [header] [099e3cf4-0303-4345-ae13-df720aac9ce4] Referrer-Policy: same-origin
D: 2023-01-26 15:17:15:030(LogBuilder.kt:38) .logHttp()[Network, response] [header] [099e3cf4-0303-4345-ae13-df720aac9ce4] Strict-Transport-Security: max-age=63072000; includeSubDomains; preload
D: 2023-01-26 15:17:15:031(LogBuilder.kt:38) .logHttp()[Network, response] [header] [099e3cf4-0303-4345-ae13-df720aac9ce4] Vary: Origin
D: 2023-01-26 15:17:15:032(LogBuilder.kt:38) .logHttp()[Network, response] [header] [099e3cf4-0303-4345-ae13-df720aac9ce4] Www-Authenticate: Basic realm="ocis.xxxxx.xxx"
D: 2023-01-26 15:17:15:033(LogBuilder.kt:38) .logHttp()[Network, response] [header] [099e3cf4-0303-4345-ae13-df720aac9ce4] Www-Authenticate: Bearer realm="ocis.xxxxx.xxx"
D: 2023-01-26 15:17:15:033(LogBuilder.kt:38) .logHttp()[Network, response] [header] [099e3cf4-0303-4345-ae13-df720aac9ce4] X-Content-Type-Options: nosniff
D: 2023-01-26 15:17:15:034(LogBuilder.kt:38) .logHttp()[Network, response] [header] [099e3cf4-0303-4345-ae13-df720aac9ce4] X-Frame-Options: allow-from https:xxxxx.xxx
D: 2023-01-26 15:17:15:035(LogBuilder.kt:38) .logHttp()[Network, response] [header] [099e3cf4-0303-4345-ae13-df720aac9ce4] X-Robots-Tag: none,noarchive,nosnippet,notranslate,noimageindex,
D: 2023-01-26 15:17:15:036(LogBuilder.kt:38) .logHttp()[Network, response] [header] [099e3cf4-0303-4345-ae13-df720aac9ce4] X-Xss-Protection: 1; mode=block
D: 2023-01-26 15:17:15:037(LogBuilder.kt:38) .logHttp()[Network, response] [body] [099e3cf4-0303-4345-ae13-df720aac9ce4] Length: 0 byte body
D: 2023-01-26 15:17:15:038(LogBuilder.kt:38) .logHttp()[Network, response] [body] [099e3cf4-0303-4345-ae13-df720aac9ce4] Type: null
D: 2023-01-26 15:17:15:038(LogBuilder.kt:38) .logHttp()[Network, response] [body] [099e3cf4-0303-4345-ae13-df720aac9ce4] --> Body start for response
D: 2023-01-26 15:17:15:039(LogBuilder.kt:38) .logHttp()[Network, response] [body] [099e3cf4-0303-4345-ae13-df720aac9ce4] <-- Body end for response -- Binary -- Omitted: 0 bytes
D: 2023-01-26 15:17:15:041(CheckPathExistenceRemoteOperation.kt:70) .run()Existence check for https://ocis.xxxxx.xxx/remote.php/dav/files/ finished with HTTP status 401(FAIL)
D: 2023-01-26 15:17:15:042(ViewModelExt.kt:64) $runUseCaseWithResult.invokeSuspend()Use case executed: GetServerInfoAsyncUseCase with result: Success(data=ServerInfo(ownCloudVersion=10.11.0.0, baseUrl=https://ocis.xxxxx.xxx, authenticationMethod=bearer, isSecureConnection=true))
D: 2023-01-26 15:17:15:049(LoginActivity.kt:433) .startOIDCOauthorization$lambda-35()Service discovery: OIDCServerConfiguration(authorizationEndpoint=https://auth.xxxxx.xxx/signin/v1/identifier/_/authorize, checkSessionIframe=https://auth.xxxxx.xxx/konnect/v1/session/check-session.html, endSessionEndpoint=https://auth.xxxxx.xxx/signin/v1/identifier/_/endsession, issuer=https://auth.xxxxx.xxx, registrationEndpoint=null, responseTypesSupported=[id_token token, id_token, code id_token, code id_token token], scopesSupported=[openid, offline_access, profile, email, LibgreGraph.UUID, LibreGraph.RawSub], tokenEndpoint=https://auth.xxxxx.xxx/konnect/v1/token, tokenEndpointAuthMethodsSupported=[client_secret_basic, none], userInfoEndpoint=https://auth.xxxxx.xxx/konnect/v1/userinfo)
D: 2023-01-26 15:17:15:050(LoginActivity.kt:497) .performGetAuthorizationCodeRequest()A browser should be opened now to authenticate this user.
D: 2023-01-26 15:17:15:052(OwnCloudClient.java:128) .saveExecuteHttpMethod()Executing in request with id dccac2cf-c388-4b05-a96b-e4aad67d10db
D: 2023-01-26 15:17:15:054(LogBuilder.kt:38) .logHttp()[Network, request] [info] [dccac2cf-c388-4b05-a96b-e4aad67d10db] Type: GET URL: https://ocis.xxxxx.xxx/.well-known/openid-configuration
D: 2023-01-26 15:17:15:056(LogBuilder.kt:38) .logHttp()[Network, request] [header] [dccac2cf-c388-4b05-a96b-e4aad67d10db] OCS-APIREQUEST: true
D: 2023-01-26 15:17:15:056(LogBuilder.kt:38) .logHttp()[Network, request] [header] [dccac2cf-c388-4b05-a96b-e4aad67d10db] X-Request-ID: dccac2cf-c388-4b05-a96b-e4aad67d10db
D: 2023-01-26 15:17:15:057(LogBuilder.kt:38) .logHttp()[Network, request] [header] [dccac2cf-c388-4b05-a96b-e4aad67d10db] User-Agent: Mozilla/5.0 (Android) ownCloud-android/3.0.1
D: 2023-01-26 15:17:15:058(LogBuilder.kt:38) .logHttp()[Network, request] [header] [dccac2cf-c388-4b05-a96b-e4aad67d10db] Accept-Encoding: identity
D: 2023-01-26 15:17:15:059(LogBuilder.kt:38) .logHttp()[Network, request] [header] [dccac2cf-c388-4b05-a96b-e4aad67d10db] Host: ocis.xxxxx.xxx
D: 2023-01-26 15:17:15:059(LogBuilder.kt:38) .logHttp()[Network, request] [header] [dccac2cf-c388-4b05-a96b-e4aad67d10db] Connection: Keep-Alive
D: 2023-01-26 15:17:15:060(LogBuilder.kt:38) .logHttp()[Network, request] [body] [dccac2cf-c388-4b05-a96b-e4aad67d10db] Empty body
D: 2023-01-26 15:17:15:073(LogBuilder.kt:38) .logHttp()[Network, response] [info] [dccac2cf-c388-4b05-a96b-e4aad67d10db] Code: 200  Message: OK IsSuccessful: true
D: 2023-01-26 15:17:15:074(LogBuilder.kt:38) .logHttp()[Network, response] [header] [dccac2cf-c388-4b05-a96b-e4aad67d10db] Access-Control-Allow-Credentials: true
D: 2023-01-26 15:17:15:074(LogBuilder.kt:38) .logHttp()[Network, response] [header] [dccac2cf-c388-4b05-a96b-e4aad67d10db] Cache-Control: no-cache, no-store, max-age=0, must-revalidate, value
D: 2023-01-26 15:17:15:074(LogBuilder.kt:38) .logHttp()[Network, response] [header] [dccac2cf-c388-4b05-a96b-e4aad67d10db] Content-Length: 1788
D: 2023-01-26 15:17:15:075(LogBuilder.kt:38) .logHttp()[Network, response] [header] [dccac2cf-c388-4b05-a96b-e4aad67d10db] Content-Security-Policy: frame-ancestors 'none'
D: 2023-01-26 15:17:15:075(LogBuilder.kt:38) .logHttp()[Network, response] [header] [dccac2cf-c388-4b05-a96b-e4aad67d10db] Content-Type: application/json; encoding=utf-8
D: 2023-01-26 15:17:15:075(LogBuilder.kt:38) .logHttp()[Network, response] [header] [dccac2cf-c388-4b05-a96b-e4aad67d10db] Date: Thu, 26 Jan 2023 14:17:14 GMT
D: 2023-01-26 15:17:15:076(LogBuilder.kt:38) .logHttp()[Network, response] [header] [dccac2cf-c388-4b05-a96b-e4aad67d10db] Expires: Thu, 01 Jan 1970 00:00:00 GMT
D: 2023-01-26 15:17:15:076(LogBuilder.kt:38) .logHttp()[Network, response] [header] [dccac2cf-c388-4b05-a96b-e4aad67d10db] Last-Modified: Thu, 26 Jan 2023 14:17:14 GMT
D: 2023-01-26 15:17:15:078(LogBuilder.kt:38) .logHttp()[Network, response] [header] [dccac2cf-c388-4b05-a96b-e4aad67d10db] Permissions-Policy: camera 'none'; geolocation 'none'; microphone 'none'; payment 'none'; usb 'none'; vr 'none';
D: 2023-01-26 15:17:15:079(LogBuilder.kt:38) .logHttp()[Network, response] [header] [dccac2cf-c388-4b05-a96b-e4aad67d10db] Referrer-Policy: same-origin
D: 2023-01-26 15:17:15:079(LogBuilder.kt:38) .logHttp()[Network, response] [header] [dccac2cf-c388-4b05-a96b-e4aad67d10db] Strict-Transport-Security: max-age=63072000; includeSubDomains; preload
D: 2023-01-26 15:17:15:079(LogBuilder.kt:38) .logHttp()[Network, response] [header] [dccac2cf-c388-4b05-a96b-e4aad67d10db] Vary: Origin
D: 2023-01-26 15:17:15:080(LogBuilder.kt:38) .logHttp()[Network, response] [header] [dccac2cf-c388-4b05-a96b-e4aad67d10db] X-Content-Type-Options: nosniff
D: 2023-01-26 15:17:15:080(LogBuilder.kt:38) .logHttp()[Network, response] [header] [dccac2cf-c388-4b05-a96b-e4aad67d10db] X-Frame-Options: allow-from https:xxxxx.xxx
D: 2023-01-26 15:17:15:080(LogBuilder.kt:38) .logHttp()[Network, response] [header] [dccac2cf-c388-4b05-a96b-e4aad67d10db] X-Idp-Version: 2.0.0-beta.5
D: 2023-01-26 15:17:15:081(LogBuilder.kt:38) .logHttp()[Network, response] [header] [dccac2cf-c388-4b05-a96b-e4aad67d10db] X-Robots-Tag: none,noarchive,nosnippet,notranslate,noimageindex,
D: 2023-01-26 15:17:15:081(LogBuilder.kt:38) .logHttp()[Network, response] [header] [dccac2cf-c388-4b05-a96b-e4aad67d10db] X-Xss-Protection: 1; mode=block
D: 2023-01-26 15:17:15:082(LogBuilder.kt:38) .logHttp()[Network, response] [body] [dccac2cf-c388-4b05-a96b-e4aad67d10db] Length: 1788 byte body
D: 2023-01-26 15:17:15:082(LogBuilder.kt:38) .logHttp()[Network, response] [body] [dccac2cf-c388-4b05-a96b-e4aad67d10db] Type: application/json; encoding=utf-8
V: 2023-01-26 15:17:15:083(MainApp.kt:142) $onCreate.onActivityPaused()LoginActivity onPause() ending
D: 2023-01-26 15:17:15:083(LogBuilder.kt:38) .logHttp()[Network, response] [body] [dccac2cf-c388-4b05-a96b-e4aad67d10db] --> Body start for response
D: 2023-01-26 15:17:15:084(LogBuilder.kt:38) .logHttp()[Network, response] [body] [dccac2cf-c388-4b05-a96b-e4aad67d10db] {
  "issuer": "https://auth.xxxxx.xxx",
  "authorization_endpoint": "https://auth.xxxxx.xxx/signin/v1/identifier/_/authorize",
  "token_endpoint": "https://auth.xxxxx.xxx/konnect/v1/token",
  "userinfo_endpoint": "https://auth.xxxxx.xxx/konnect/v1/userinfo",
  "end_session_endpoint": "https://auth.xxxxx.xxx/signin/v1/identifier/_/endsession",
  "check_session_iframe": "https://auth.xxxxx.xxx/konnect/v1/session/check-session.html",
  "jwks_uri": "https://auth.xxxxx.xxx/konnect/v1/jwks.json",
  "scopes_supported": [
    "openid",
    "offline_access",
    "profile",
    "email",
    "LibgreGraph.UUID",
    "LibreGraph.RawSub"
  ],
  "response_types_supported": [
    "id_token token",
    "id_token",
    "code id_token",
    "code id_token token"
  ],
  "subject_types_supported": [
    "public"
  ],
  "id_token_signing_alg_values_supported": [
    "PS256",
    "PS384",
    "PS512",
    "RS256",
    "RS384",
    "RS512"
  ],
  "userinfo_signing_alg_values_supported": [
    "PS256",
    "PS384",
    
D: 2023-01-26 15:17:15:084(LogBuilder.kt:38) .logHttp()[Network, response] [body] [dccac2cf-c388-4b05-a96b-e4aad67d10db] <-- Body end for response -- Omitted: 764 bytes
D: 2023-01-26 15:17:15:085(GetOIDCDiscoveryRemoteOperation.kt:64) .run()Successful response {
  "issuer": "https://auth.xxxxx.xxx",
  "authorization_endpoint": "https://auth.xxxxx.xxx/signin/v1/identifier/_/authorize",
  "token_endpoint": "https://auth.xxxxx.xxx/konnect/v1/token",
  "userinfo_endpoint": "https://auth.xxxxx.xxx/konnect/v1/userinfo",
  "end_session_endpoint": "https://auth.xxxxx.xxx/signin/v1/identifier/_/endsession",
  "check_session_iframe": "https://auth.xxxxx.xxx/konnect/v1/session/check-session.html",
  "jwks_uri": "https://auth.xxxxx.xxx/konnect/v1/jwks.json",
  "scopes_supported": [
    "openid",
    "offline_access",
    "profile",
    "email",
    "LibgreGraph.UUID",
    "LibreGraph.RawSub"
  ],
  "response_types_supported": [
    "id_token token",
    "id_token",
    "code id_token",
    "code id_token token"
  ],
  "subject_types_supported": [
    "public"
  ],
  "id_token_signing_alg_values_supported": [
    "PS256",
    "PS384",
    "PS512",
    "RS256",
    "RS384",
    "RS512"
  ],
  "userinfo_signing_alg_values_supported": [
    "PS256",
    "PS384",
    "PS512",
    "RS256",
    "RS384",
    "RS512"
  ],
  "request_object_signing_alg_values_supported": [
    "ES256",
    "ES384",
    "ES512",
    "RS256",
    "RS384",
    "RS512",
    "PS256",
    "PS384",
    "PS512",
    "none",
    "EdDSA"
  ],
  "token_endpoint_auth_methods_supported": [
    "client_secret_basic",
    "none"
  ],
  "token_endpoint_auth_signing_alg_values_supported": [
    "PS256",
    "PS384",
    "PS512",
    "RS256",
    "RS384",
    "RS512"
  ],
  "claims_parameter_supported": true,
  "claims_supported": [
    "iss",
    "sub",
    "aud",
    "exp",
    "iat",
    "name",
    "family_name",
    "given_name",
    "email",
    "email_verified"
  ],
  "request_parameter_supported": true,
  "request_uri_parameter_supported": false
}

D: 2023-01-26 15:17:15:087(GetOIDCDiscoveryRemoteOperation.kt:70) .run()Get OIDC Discovery completed and parsed to [OIDCDiscoveryResponse(authorization_endpoint=https://auth.xxxxx.xxx/signin/v1/identifier/_/authorize, check_session_iframe=https://auth.xxxxx.xxx/konnect/v1/session/check-session.html, end_session_endpoint=https://auth.xxxxx.xxx/signin/v1/identifier/_/endsession, issuer=https://auth.xxxxx.xxx, registration_endpoint=null, response_types_supported=[id_token token, id_token, code id_token, code id_token token], scopes_supported=[openid, offline_access, profile, email, LibgreGraph.UUID, LibreGraph.RawSub], token_endpoint=https://auth.xxxxx.xxx/konnect/v1/token, token_endpoint_auth_methods_supported=[client_secret_basic, none], userinfo_endpoint=https://auth.xxxxx.xxx/konnect/v1/userinfo)]
D: 2023-01-26 15:17:15:091(ViewModelExt.kt:64) $runUseCaseWithResult.invokeSuspend()Use case executed: OIDCDiscoveryUseCase with result: Success(data=OIDCServerConfiguration(authorizationEndpoint=https://auth.xxxxx.xxx/signin/v1/identifier/_/authorize, checkSessionIframe=https://auth.xxxxx.xxx/konnect/v1/session/check-session.html, endSessionEndpoint=https://auth.xxxxx.xxx/signin/v1/identifier/_/endsession, issuer=https://auth.xxxxx.xxx, registrationEndpoint=null, responseTypesSupported=[id_token token, id_token, code id_token, code id_token token], scopesSupported=[openid, offline_access, profile, email, LibgreGraph.UUID, LibreGraph.RawSub], tokenEndpoint=https://auth.xxxxx.xxx/konnect/v1/token, tokenEndpointAuthMethodsSupported=[client_secret_basic, none], userInfoEndpoint=https://auth.xxxxx.xxx/konnect/v1/userinfo))
D: 2023-01-26 15:17:15:095(LoginActivity.kt:433) .startOIDCOauthorization$lambda-35()Service discovery: OIDCServerConfiguration(authorizationEndpoint=https://auth.xxxxx.xxx/signin/v1/identifier/_/authorize, checkSessionIframe=https://auth.xxxxx.xxx/konnect/v1/session/check-session.html, endSessionEndpoint=https://auth.xxxxx.xxx/signin/v1/identifier/_/endsession, issuer=https://auth.xxxxx.xxx, registrationEndpoint=null, responseTypesSupported=[id_token token, id_token, code id_token, code id_token token], scopesSupported=[openid, offline_access, profile, email, LibgreGraph.UUID, LibreGraph.RawSub], tokenEndpoint=https://auth.xxxxx.xxx/konnect/v1/token, tokenEndpointAuthMethodsSupported=[client_secret_basic, none], userInfoEndpoint=https://auth.xxxxx.xxx/konnect/v1/userinfo)
D: 2023-01-26 15:17:15:095(LoginActivity.kt:497) .performGetAuthorizationCodeRequest()A browser should be opened now to authenticate this user.
D: 2023-01-26 15:17:15:113(LoginActivity.kt:433) .startOIDCOauthorization$lambda-35()Service discovery: OIDCServerConfiguration(authorizationEndpoint=https://auth.xxxxx.xxx/signin/v1/identifier/_/authorize, checkSessionIframe=https://auth.xxxxx.xxx/konnect/v1/session/check-session.html, endSessionEndpoint=https://auth.xxxxx.xxx/signin/v1/identifier/_/endsession, issuer=https://auth.xxxxx.xxx, registrationEndpoint=null, responseTypesSupported=[id_token token, id_token, code id_token, code id_token token], scopesSupported=[openid, offline_access, profile, email, LibgreGraph.UUID, LibreGraph.RawSub], tokenEndpoint=https://auth.xxxxx.xxx/konnect/v1/token, tokenEndpointAuthMethodsSupported=[client_secret_basic, none], userInfoEndpoint=https://auth.xxxxx.xxx/konnect/v1/userinfo)
D: 2023-01-26 15:17:15:114(LoginActivity.kt:497) .performGetAuthorizationCodeRequest()A browser should be opened now to authenticate this user.
V: 2023-01-26 15:17:15:756(MainApp.kt:146) $onCreate.onActivityStopped()LoginActivity onStop() ending
```

**Comment by C8opmBM** [comment]
One thing I noted it won't work if you're not on your home network (for mobile). Maybe that could be your issue?
Sorry, don't have anything more here. Should work.

As for my issue with the desktop app and SWAG/NGINX, I fixed it following the snippet from @helgeklein caddy, and adapted it to ngix. So all 3 work now for me (web, android and desktop).

```
        if ($args ~* (.*)(&prompt=select_account%20consent)(.*)) {
            set $args $1$3;
            rewrite ^(.*)$ $1;
        }
```

**Comment by alceasan** [comment]
Well, this is even more strange. After trying it with the mobile network, now I'm getting a "Server not found" message in the woncloud app. It doesn't even try to open a browser for authentication in Authelia.

I verified it wasn't a network problem connecting from the mobile network to the web client without issues. This is the log:

```
D: 2023-01-27 09:23:35:200(OwnCloudClient.java:90) .<init>()#10Creating OwnCloudClient
D: 2023-01-27 09:23:35:203(OwnCloudClient.java:128) .saveExecuteHttpMethod()Executing in request with id da0f8374-ad58-4867-b9f9-5b7281f8ec4b
D: 2023-01-27 09:23:35:392(LogBuilder.kt:38) .logHttp()[Network, request] [info] [da0f8374-ad58-4867-b9f9-5b7281f8ec4b] Type: GET URL: https://ocis.xxxxx.xxx/status.php
D: 2023-01-27 09:23:35:394(LogBuilder.kt:38) .logHttp()[Network, request] [header] [da0f8374-ad58-4867-b9f9-5b7281f8ec4b] X-Request-ID: da0f8374-ad58-4867-b9f9-5b7281f8ec4b
D: 2023-01-27 09:23:35:395(LogBuilder.kt:38) .logHttp()[Network, request] [header] [da0f8374-ad58-4867-b9f9-5b7281f8ec4b] User-Agent: Mozilla/5.0 (Android) ownCloud-android/3.0.2
D: 2023-01-27 09:23:35:395(LogBuilder.kt:38) .logHttp()[Network, request] [header] [da0f8374-ad58-4867-b9f9-5b7281f8ec4b] Accept-Encoding: identity
D: 2023-01-27 09:23:35:396(LogBuilder.kt:38) .logHttp()[Network, request] [header] [da0f8374-ad58-4867-b9f9-5b7281f8ec4b] Host: ocis.xxxxx.xxx
D: 2023-01-27 09:23:35:397(LogBuilder.kt:38) .logHttp()[Network, request] [header] [da0f8374-ad58-4867-b9f9-5b7281f8ec4b] Connection: Keep-Alive
D: 2023-01-27 09:23:35:398(LogBuilder.kt:38) .logHttp()[Network, request] [body] [da0f8374-ad58-4867-b9f9-5b7281f8ec4b] Empty body
D: 2023-01-27 09:23:35:585(LogBuilder.kt:38) .logHttp()[Network, response] [info] [da0f8374-ad58-4867-b9f9-5b7281f8ec4b] Code: 401  Message: Unauthorized IsSuccessful: false
D: 2023-01-27 09:23:35:589(LogBuilder.kt:38) .logHttp()[Network, response] [header] [da0f8374-ad58-4867-b9f9-5b7281f8ec4b] Access-Control-Allow-Credentials: true
D: 2023-01-27 09:23:35:590(LogBuilder.kt:38) .logHttp()[Network, response] [header] [da0f8374-ad58-4867-b9f9-5b7281f8ec4b] Content-Length: 114
D: 2023-01-27 09:23:35:591(LogBuilder.kt:38) .logHttp()[Network, response] [header] [da0f8374-ad58-4867-b9f9-5b7281f8ec4b] Content-Type: text/html; charset=utf-8
D: 2023-01-27 09:23:35:592(LogBuilder.kt:38) .logHttp()[Network, response] [header] [da0f8374-ad58-4867-b9f9-5b7281f8ec4b] Date: Fri, 27 Jan 2023 08:23:34 GMT
D: 2023-01-27 09:23:35:593(LogBuilder.kt:38) .logHttp()[Network, response] [header] [da0f8374-ad58-4867-b9f9-5b7281f8ec4b] Location: https://auth.xxxxx.xxx/?rd=https%3A%2F%2Focis.xxxxx.xxx%2Fstatus.php&rm=GET
D: 2023-01-27 09:23:35:594(LogBuilder.kt:38) .logHttp()[Network, response] [header] [da0f8374-ad58-4867-b9f9-5b7281f8ec4b] Permissions-Policy: camera 'none'; geolocation 'none'; microphone 'none'; payment 'none'; usb 'none'; vr 'none';
D: 2023-01-27 09:23:35:595(LogBuilder.kt:38) .logHttp()[Network, response] [header] [da0f8374-ad58-4867-b9f9-5b7281f8ec4b] Referrer-Policy: same-origin
D: 2023-01-27 09:23:35:596(LogBuilder.kt:38) .logHttp()[Network, response] [header] [da0f8374-ad58-4867-b9f9-5b7281f8ec4b] Set-Cookie: authelia_session=^h^!GJvDNgaceqr6AJm12bEvE_-Komcl; expires=Fri, 27 Jan 2023 09:23:35 GMT; domain=xxxxx.xxx; path=/; HttpOnly; secure; SameSite=Lax
D: 2023-01-27 09:23:35:597(LogBuilder.kt:38) .logHttp()[Network, response] [header] [da0f8374-ad58-4867-b9f9-5b7281f8ec4b] Strict-Transport-Security: max-age=63072000; includeSubDomains; preload
D: 2023-01-27 09:23:35:598(LogBuilder.kt:38) .logHttp()[Network, response] [header] [da0f8374-ad58-4867-b9f9-5b7281f8ec4b] Vary: Origin
D: 2023-01-27 09:23:35:598(LogBuilder.kt:38) .logHttp()[Network, response] [header] [da0f8374-ad58-4867-b9f9-5b7281f8ec4b] X-Content-Type-Options: nosniff
D: 2023-01-27 09:23:35:599(LogBuilder.kt:38) .logHttp()[Network, response] [header] [da0f8374-ad58-4867-b9f9-5b7281f8ec4b] X-Frame-Options: allow-from https:xxxxx.xxx
D: 2023-01-27 09:23:35:600(LogBuilder.kt:38) .logHttp()[Network, response] [header] [da0f8374-ad58-4867-b9f9-5b7281f8ec4b] X-Robots-Tag: none,noarchive,nosnippet,notranslate,noimageindex,
D: 2023-01-27 09:23:35:601(LogBuilder.kt:38) .logHttp()[Network, response] [header] [da0f8374-ad58-4867-b9f9-5b7281f8ec4b] X-Xss-Protection: 1; mode=block
D: 2023-01-27 09:23:35:601(LogBuilder.kt:38) .logHttp()[Network, response] [body] [da0f8374-ad58-4867-b9f9-5b7281f8ec4b] Length: 114 byte body
D: 2023-01-27 09:23:35:602(LogBuilder.kt:38) .logHttp()[Network, response] [body] [da0f8374-ad58-4867-b9f9-5b7281f8ec4b] Type: text/html; charset=utf-8
D: 2023-01-27 09:23:35:603(LogBuilder.kt:38) .logHttp()[Network, response] [body] [da0f8374-ad58-4867-b9f9-5b7281f8ec4b] --> Body start for response
D: 2023-01-27 09:23:35:604(LogBuilder.kt:38) .logHttp()[Network, response] [body] [da0f8374-ad58-4867-b9f9-5b7281f8ec4b] <a href="https://auth.xxxxx.xxx/?rd=https%3A%2F%2Focis.xxxxx.xxx%2Fstatus.php&amp;rm=GET">401 Unauthorized</a>
D: 2023-01-27 09:23:35:605(LogBuilder.kt:38) .logHttp()[Network, response] [body] [da0f8374-ad58-4867-b9f9-5b7281f8ec4b] <-- Body end for response -- Omitted: 0 bytes
E: 2023-01-27 09:23:35:607(RemoteOperationResult.java:119) .<init>()---------------------------------
Create RemoteOperationResult from exception.
 Message: NullPointerException: Attempt to invoke virtual method 'java.lang.String com.owncloud.android.lib.resources.status.RemoteServerInfo.getBaseUrl()' on a null object reference
 Stacktrace: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String com.owncloud.android.lib.resources.status.RemoteServerInfo.getBaseUrl()' on a null object reference
	at com.owncloud.android.lib.resources.status.GetRemoteStatusOperation.tryToConnect(GetRemoteStatusOperation.kt:63)
	at com.owncloud.android.lib.resources.status.GetRemoteStatusOperation.run(GetRemoteStatusOperation.kt:49)
	at com.owncloud.android.lib.common.operations.RemoteOperation.runOperation(RemoteOperation.java:264)
	at com.owncloud.android.lib.common.operations.RemoteOperation.execute(RemoteOperation.java:229)
	at com.owncloud.android.lib.resources.status.services.implementation.OCServerInfoService.getRemoteStatus(OCServerInfoService.kt:53)
	at com.owncloud.android.data.server.datasources.implementation.OCRemoteServerInfoDataSource.getRemoteStatus(OCRemoteServerInfoDataSource.kt:77)
	at com.owncloud.android.data.server.datasources.implementation.OCRemoteServerInfoDataSource.getServerInfo(OCRemoteServerInfoDataSource.kt:92)
	at com.owncloud.android.data.server.repository.OCServerInfoRepository.getServerInfo(OCServerInfoRepository.kt:31)
	at com.owncloud.android.domain.server.usecases.GetServerInfoAsyncUseCase.run(GetServerInfoAsyncUseCase.kt:30)
	at com.owncloud.android.domain.server.usecases.GetServerInfoAsyncUseCase.run(GetServerInfoAsyncUseCase.kt:26)
	at com.owncloud.android.domain.BaseUseCaseWithResult.execute(BaseUseCaseWithResult.kt:30)
	at com.owncloud.android.extensions.ViewModelExt$runUseCaseWithResult$1.invokeSuspend(ViewModelExt.kt:62)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at kotlinx.coroutines.internal.LimitedDispatcher.run(LimitedDispatcher.kt:42)
	at kotlinx.coroutines.scheduling.TaskImpl.run(Tasks.kt:95)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:570)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:749)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:677)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:664)

---------------------------------
D: 2023-01-27 09:23:35:608(ViewModelExt.kt:64) $runUseCaseWithResult.invokeSuspend()Use case executed: GetServerInfoAsyncUseCase with result: Error(throwable=com.owncloud.android.domain.exceptions.ServerNotReachableException)
V: 2023-01-27 09:23:37:904(MainApp.kt:142) $onCreate.onActivityPaused()LoginActivity onPause() ending
D: 2023-01-27 09:23:37:920(MainApp.kt:102) $onCreate.onActivityCreated()SettingsActivity onCreate(Bundle) starting
V: 2023-01-27 09:23:37:929(MainApp.kt:129) $onCreate.onActivityStarted()SettingsActivity onStart() starting
V: 2023-01-27 09:23:37:933(MainApp.kt:138) $onCreate.onActivityResumed()SettingsActivity onResume() starting
V: 2023-01-27 09:23:38:465(MainApp.kt:146) $onCreate.onActivityStopped()LoginActivity onStop() ending
V: 2023-01-27 09:23:38:466(MainApp.kt:155) $onCreate.onActivitySaveInstanceState()LoginActivity onSaveInstanceState(Bundle) starting
V: 2023-01-27 09:23:40:702(MainApp.kt:142) $onCreate.onActivityPaused()SettingsActivity onPause() ending
D: 2023-01-27 09:23:40:714(MainApp.kt:102) $onCreate.onActivityCreated()LogsListActivity onCreate(Bundle) starting
V: 2023-01-27 09:23:40:724(MainApp.kt:129) $onCreate.onActivityStarted()LogsListActivity onStart() starting
V: 2023-01-27 09:23:40:724(MainApp.kt:138) $onCreate.onActivityResumed()LogsListActivity onResume() starting
V: 2023-01-27 09:23:41:248(MainApp.kt:146) $onCreate.onActivityStopped()SettingsActivity onStop() ending
V: 2023-01-27 09:23:41:249(MainApp.kt:155) $onCreate.onActivitySaveInstanceState()SettingsActivity onSaveInstanceState(Bundle) starting
V: 2023-01-27 09:23:44:395(MainApp.kt:142) $onCreate.onActivityPaused()LogsListActivity onPause() ending
V: 2023-01-27 09:23:46:556(MainApp.kt:146) $onCreate.onActivityStopped()LogsListActivity onStop() ending
V: 2023-01-27 09:23:46:559(MainApp.kt:155) $onCreate.onActivitySaveInstanceState()LogsListActivity onSaveInstanceState(Bundle) starting
V: 2023-01-27 09:23:47:495(MainApp.kt:129) $onCreate.onActivityStarted()LogsListActivity onStart() starting
V: 2023-01-27 09:23:47:498(MainApp.kt:138) $onCreate.onActivityResumed()LogsListActivity onResume() starting
```

I also tried with the iOS app using an iPhone and having the same problems: connected with WIFI it gets stuck on the Authelia website after successful login, and the same "no server found" error on mobile network.
